tools/ci_build/github/azure-pipelines/templates/download-deps.yml

```diff
@@ -11,7 +11,7 @@ steps:
       packageType: upack
       feed: '/7424c8e4-5c62-490e-95c4-79446f31017c'
       definition: '517c4f6f-5437-4392-a70d-4f15ec5be2f0'
-      version: 1.0.25
+      version: 1.0.27
```

Comment:
what's the reason for this change?

Reply:
It is required [here](https://microsoft.sharepoint.com/teams/ONNX2/_layouts/OneNote.aspx?id=%2Fteams%2FONNX2%2FShared%20Documents%2FNotebooks%2FONNX%20Ecosystem%20Team%20Notebook&wd=target%28Development.one%7C63D3AB47-51D1-4A62-9965-66882234BD44%2FAdd%20or%20update%20a%20dependency%20in%20deps.txt%7C0E9ED71D-89D5-40FA-B05F-C0123289C591%2F%29
onenote:https://microsoft.sharepoint.com/teams/ONNX2/Shared%20Documents/Notebooks/ONNX%20Ecosystem%20Team%20Notebook/Development.one#Add%20or%20update%20a%20dependency%20in%20deps.txt&section-id={63D3AB47-51D1-4A62-9965-66882234BD44}&page-id={0E9ED71D-89D5-40FA-B05F-C0123289C591}&object-id={787EC262-79DD-0231-3E6A-B1288ECB8986}&74).